packages/unlock-js/src/__tests__/integration/walletServiceIntegration.test.js

```diff
@@ -304,14 +304,13 @@ describe.each(UnlockVersionNumbers)('Unlock %s', (unlockVersion) => {
               transactionHash = hash
             }
           )
-          key =
-            ['v10'].indexOf(publicLockVersion) !== -1
-              ? await web3Service.getKeyByTokenId(lockAddress, tokenId, chainId)
-              : await web3Service.getKeyByLockForOwner(
-                  lockAddress,
-                  keyGrantee,
-                  chainId
-                )
+          key = ['v11'].includes(publicLockVersion)
```

Comment:
@julien51 changed everywhere we call `getKeyByTokenId ` from  `v10 -> v11` based this lines on `web3service.ts` 

```
async getKeyByTokenId(lockAddress: string, tokenId: string, network: number) {
    const lockContract = await this.getLockContract(
      lockAddress,
      this.providerForNetwork(network)
    )
    if ((await lockContract.publicLockVersion()) < 11) {
      throw new Error('Only available for Lock v11+')
    }
```

Reply:
No no we should try to keep a consistebt API.

Lets use web3Service.getKeyByLockForOwner which should work for all versions of public lock please.

Reply:
I added it intentionally to this PR. It was a mistake that it was missing